src/pyfvtool/utilities.py

```diff
@@ -86,3 +88,42 @@ def FL(r):
     return FL
 
 
+
+class SignedTuple(tuple):
```

Comment:
I am getting ahead of myself, but I was thinking about performance here. Do you think it can have a large computational overhead?

Reply:
At present, this `SignedTuple` class is not used (and perhaps it will never be).
I would not worry about performance/overhead, because I think that most of the computational burden (by far) is in solving the matrix equation. Am I right?

Reply:
The linear solver will certainly be the bottleneck. However, the matrix assembly can become costly too and in a time loop it can translate to a hefty chunk of the computational time. We can always deal with it later.